src/bindgen.rs

```diff
@@ -1,24 +1,67 @@
 use emoji;
 use error::Error;
 use progressbar::Step;
-use std::process::Command;
+use std::{path, process::Command};
 use PBAR;
 
-pub fn cargo_install_wasm_bindgen(step: &Step) -> Result<(), Error> {
+#[cfg(not(target_family = "windows"))]
+static LOCAL_BINDGEN_PATH: &str = "bin/wasm-bindgen";
+
+#[cfg(target_family = "windows")]
+static LOCAL_BINDGEN_PATH: &str = "bin\\wasm-bindgen";
+
+fn local_wasm_bindgen_path_str(crate_path: &str) -> String {
+    #[cfg(not(target_family = "windows"))]
+    return format!("{}/{}", crate_path, LOCAL_BINDGEN_PATH);
+    #[cfg(target_family = "windows")]
+    return format!("{}\\{}", crate_path, LOCAL_BINDGEN_PATH);
+}
+
+pub fn wasm_bindgen_version_check(
+    crate_path: &str,
+    dep_version: &str,
+    step: &Step,
+) -> Result<bool, Error> {
+    let msg = format!("{}Checking WASM-bindgen dependency...", emoji::CHECK);
```

Comment:
not sure if this should be a step displayed to users- i think we should just do it in the install wasm-bindgen step